extension-manager-integration-test-java/src/main/java/com/exasol/extensionmanager/itest/base/AbstractVirtualSchemaExtensionIT.java

```diff
@@ -308,6 +324,30 @@ public void virtualSchemaWorks() {
         assertVirtualSchemaContent("my_VS");
     }
 
+    /**
+     * Verify that a creating a {@code VIRTUAL SCHEMA} with an existing name fails.
+     */
+    @Test
+    public void createVirtualSchemaWithDuplicateNameFails() {
+        getSetup().client().install();
+        prepareInstance();
+        createInstance("my_VS");
+        getSetup().client().assertRequestFails(() -> createInstance("my_VS"), //
+                "Virtual Schema 'my_VS' already exists", 400);
+    }
+
```

Comment:
Maybe the case-insensitive case would be sufficient ....

Reply:
You are right. This will speed up tests!